### PR TITLE
[ work in progress ] Initial rough draft for calculate capacity

### PR DESCRIPTION
**Pull request by rogeruiz**
👀 18F/cg-atlas#196

### CODE REVIEW

File: content/docs/ops/calculate-queue-capacity.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+| parser/0 (yyy)       | running | n/a | parser    | 192.168.x.x |
+| parser/1 (xxx)       | running | n/a | parser    | 192.168.x.x |
+|                      | running | n/a |           |             |
++----------------------+---------+-----+-----------+-------------+
+```
+
+Once you have the name of the VMs you can run the following commands to turn off
+the parsers. Please turn them back on after the calculations.
+
+You will first need to ssh into each parser VM you find. Let's log into the
```
Comment (cnelson) [inline content/docs/ops/calculate-queue-capacity.md:40]: Can we just simplify this to running one command?

`bosh-cli -e env -d logsearch ssh parser -- sudo /var/vcap/bosh/bin/monit stop parser`

File: content/docs/ops/calculate-queue-capacity.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Run this from within a jumpbox, preferably staging, with the parser turned off.
+This will allow for the queue to
+
+
+To calculate **memory per log bytes** take the `used_memory` using redis-cli
+
+
+
+Figure out how many logs we have in the queueo
```
Comment (cnelson) [inline content/docs/ops/calculate-queue-capacity.md:124]: Do we want to mention that you should turn the parsers back on when you are done, or is that self evident?

`bosh-cli -e env -d logsearch ssh parser -- sudo /var/vcap/bosh/bin/monit start parser`

File: content/docs/ops/runbook/troubleshooting-bosh.md [hunk 3]
```diff
@@ -33,15 +43,13 @@ documentation](https://github.com/concourse/fly#installing-from-the-concourse-ui
...
+command instead of going through the UI.
 
-```sh
-$ fly --target <YOUR_CONCOURSE_TARGET_NAME> login --concourse-url <CONCOURSE_URL> (e.g. https://ci.example.com)
-```
-You should now see that target when you issue the following command.
-```sh
-$ fly targets
+```shell
+fly -t <YOUR_CONCOURSE_TARGET_NAME> trigger-job -j jumpbox/<JOB_NAME>
```
Comment (cnelson) [inline content/docs/ops/runbook/troubleshooting-bosh.md:50]: Should we simplify this as well?  Just state that when you run this command, it prints your build number out:

`started jumpbox/<JOB_NAME> #<BUILD_NUMBER>`

and you can skip all the complexity below by then running

`fly -t <YOUR_CONCOURSE_TARGET_NAME> intercept -j jumpbox/<JOB_NAME> -b <BUILD_NUMBER>` 

and selecting the `task` container

**Comment by jmcarp** [comment]
This is going to be obsolete soon, since we're working on deprecating the redis queue for logstash persistent queues. If we decide to document capacity for the persistent queues, we'll probably want to start this from scratch. Closing.

**Pull request closed without merging**